upgrade/UPGRADE-v10.0.0.md

```diff
@@ -19,11 +19,9 @@ The first you should do is update your composer.json to have installable set on
 Pay special attention to any of your custom libraries as it's possible you will have to upgrade them too.  
 All the changes are considered as backwards incompatible.
 
-<!--- TODO change master to released version in the link  --->
 - upgrade PHP version to 8.1 ([#2430](https://github.com/shopsys/shopsys/pull/2430))
     - update `docker/php-fpm/Dockerfile` and `scripts/install-docker-wsl-debian.sh` according to [project-base-diff](https://github.com/shopsys/project-base/commit/8ddaab98a3d3392c13735541505202cb63dfa5ce)
-        <!--- TODO regenerate the gist during release  --->
-    - apply all [composer.json changes](https://gist.github.com/ShopsysBot/85fd56266c93a75b575c24499463f0d9/revisions?diff=split) from this version at once
+    - apply all [composer.json changes](https://gist.github.com/ShopsysBot/b56c6679bf6a0ce96b4bb91135521c46/revisions?diff=split) from this version at once
     - update your dependencies with `composer update` after you set `platform.php` in `composer.json` to the required version
```

Comment:
in this file is still one unprocessed `#project-base-diff` placeholder, can you check it please?

Reply:
ah, good catch, I totally forgot about this :+1:

Reply:
fixed in 402ec5987